pkgs/development/tools/misc/texinfo/4.13a.nix

```diff
@@ -1,4 +1,11 @@
-{ stdenv, fetchurl, texinfo, ncurses, xz, lib }:
+{
+  stdenv,
+  fetchurl,
+  texinfo,
+  ncurses,
+  xz,
+  lib,
+}:
```

Comment:
```
{ stdenv
, fetchurl
, ...
````
format is usually preferred. But I would actually keep the original line. After all, it is rather short and unlikely to ever change, given this is a legacy version.

Reply:
[See my comment above](https://github.com/NixOS/nixpkgs/pull/289690#discussion_r1623569633) about nixfmt

Reply:
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#code-conventions

Reply:
Possibly I misunderstood the meaning and impact of [RFC 0166](https://github.com/NixOS/rfcs/pull/166), from which I took that [`NixOS/nixfmt`](http://github.com/NixOS/nixfmt) should be used moving forward. From this point of view the [**Code Conventions** in `CONTRIBUTING.md`](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#code-conventions) are outdated.

What's your take on this, @oxij?